docs/docs/dev_docs/debugging/main.md

```diff
@@ -18,19 +18,25 @@ use dep::aztec::oracle::debug_log::{ debug_log };
 ```
 
 **Write log**
-Write `debug_log()` in the appropriate place in your contract. 
+Write `debug_log()` in the appropriate place in your contract.
```

Comment:
We should include documentation on the various debug types, more often than not people want to just log a field element. This is not possible with the single debug log without using the templating syntax